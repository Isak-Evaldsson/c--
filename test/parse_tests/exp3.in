6 + (5 * 7 - test)