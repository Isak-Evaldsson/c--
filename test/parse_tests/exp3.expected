BinaryExpr: +
    ConstExpr: 6
    BinaryExpr: -
        BinaryExpr: *
            ConstExpr: 5
            ConstExpr: 7
        VarUseExpr: test
